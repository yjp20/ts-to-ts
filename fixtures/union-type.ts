/** @model */
type Status = "active" | "inactive";
