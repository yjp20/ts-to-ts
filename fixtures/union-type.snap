
<<<<<<<<<<< typescript <<<<<<<<<<<<
/** @model */
type Status = "active" | "inactive";
===================================
// union-type.ts
alias Status = "active" | "inactive";
>>>>>>>>>>> typespec >>>>>>>>>>>>>>